akka-testkit-typed/src/main/scala/akka/testkit/typed/internal/TestProbeImpl.scala

```diff
@@ -41,20 +40,17 @@ private[akka] object TestProbeImpl {
 }
 
 @InternalApi
-private[akka] final class TestProbeImpl[M](name: String, system: ActorSystem[_]) extends JavaTestProbe[M] with ScalaTestProbe[M] {
+private[akka] final class TestProbeImpl[M](name: String, system: ActorSystem[_]) extends JavaTestProbe[M] with ScalaTestProbe[M] with Within {
```

Comment:
The shared trait is for sharing between the TestProbe and the ActorTestKit

Reply:
Do we really need to have it in both? Maybe just TestProbe?

Reply:
Would be nice on both IMO, esp if testkit becomes a value.

Reply:
@patriknw may have an opinion

Reply:
it's not needed for that reason, because ActorTestKit should not need `Within`. It can be a nice way to structure the classes anyway.